Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -21297,13 +21324,16 @@
 				FD2DBF0F12B048A300ED98C6 /* AudioToolbox.framework */,
 				A1763F401E20586E001D58DE /* CFNetwork.framework */,
 				AA5F3B9016CC5BEB00455EB0 /* CoreFoundation.framework */,
+				948FD0332B4C9D0600043FEB /* CryptoKit.framework */,
+				948FD0432B4C9D8800043FEB /* CryptoKit.framework */,
 				A15E6BF01E212A6A0080AF34 /* Foundation.framework */,
 				1AB33DA412551E320024457A /* IOKit.framework */,
 				F8216299029F4FB501000131 /* JavaScriptCore.framework */,
 				C0E2F3252412BF29009C73EC /* libAccessibility.tbd */,
 				1C8D26D022C09CDE00D125F3 /* libcompression.tbd */,
 				93F1D31A0558CC5C00821BC0 /* libicucore.dylib */,
 				1CFAE3220A6D6A3F0032593D /* libobjc.dylib */,
+				94EDD04C2B59E8B3001A3AC4 /* libswiftCore.tbd */,
```

Comment:
I don't think we need a dependency on libswift added in Xcode's UI if we also specify the dependency in OTHER_LDFLAGS.